neurodsp/plts/combined.py

```diff
@@ -0,0 +1,63 @@
+"""Plotting functions for plots with combined panels."""
+
+import matplotlib.pyplot as plt
+
+from neurodsp.spectral import compute_spectrum
+from neurodsp.spectral.utils import trim_spectrum
+from neurodsp.plts.spectral import plot_power_spectra
+from neurodsp.plts.time_series import plot_time_series
+from neurodsp.plts.utils import savefig
+
+###################################################################################################
+###################################################################################################
+
+@savefig
+def plot_timeseries_and_spectrum(times, sig, fs, f_range=None, spectrum_kwargs=None,
```

Comment:
Could `times` be computed in this function, since `fs` is passed, or do you think it should be an arg? Actually since `plot_time_series` requires it too, this makes sense. But this highlights why I never use plot_time_series and opt for plt.plot(sig) instead, maybe I'm just lazy lol.

Reply:
Also, the func could be added to init for lazier imports.

Reply:
Yeh, that's a good point - this function doesn't totally _need_ `times`, it can recompute from sig length and fs, so I guess the main question is whether there is any reason to try and map the API here to match plot_timeseries (?)

It didn't seem to me there was a strong reason to keep times - so I've pushed an update to drop it, and recompute. I also added a convenience argument to restrict the plotted time series range, and an option to update the start value (since the only real benefit of having `times` is that recomputing assume a time range of 0-siglength, whereas providing a specific times vector allows for anything). Do you think this is better?

In terms of using plt.plot instead of plot_time_series - they are sorta the same in the end hahaha, but (just double-checking you know) you can pass None as times and then you get it plotted by sample, instead of having to manually compute times!

Also - I tried adding to __init__, but the tests failed for a circular import, which I didn't try and chase out. Should we try and address that do you think?

Reply:
Cool! I like the change. Another optional is to make `times` an optional kwarg. But using `start_val` also works. Up to you, the are the same to me.

I forget / didn't know about passing None to `plot_time_series`. I may use it more now! Especially since it removes the need for styling, etc.

Hmm, yeah that's odd. I spent a couple minutes looking at it and it's not obvious. Seems like something to do with these interactions:
```
compute_spectrum -> filter_signal_fir -> plot_filter_properties
compute_spectrum -> plot_timeseries_and_spectrum
plot_timeseries_and_spectrum and plot_filter_properties in __init__?
```
A quick/lazy solution is to drop the `neurodsp.spectral` imports inside the body of `plot_timeseries_and_spectrum`.